source/functions/BEMIO/radiationIRF.m

```diff
@@ -68,7 +68,7 @@
 % Calculate the infinite frequency added mass
 ra_Ainf_temp = zeros(length(hydro.w),1);                                    %Initialize the variable
 [~,F] = size(hydro);                                                        %Last data set in
-if strcmp(hydro(F).code,'WAMIT')==0
+if isfield(hydro,'Ainf') == 0
```

Comment:
@salhus So this change will not update the infinite frequency added mass value for readNEMOH since this is populated in the normalizeBEM function. We either need to add a check here to see if Ainf is empty or the code is not WAMIT to implement the fourier transform to estimate the infinite frequency added mass. Or remove a line in normalizeBEM to not pre-populate the infinite frequency added mass.

Reply:
@nathanmtom 
Sorry I didnt realize normalizeBEM was doing that. I think I like the approach where normalizeBEM is not prepopulating the infinite added mass. This would make future debugging easier.